src/apps/apploader/app.js

```diff
@@ -534,25 +534,17 @@ define(function(require) {
 						appLinks = _.keys(monster.config.whitelabel.appLinks),
 						updateUserApps = false,
 						isAppInstalled = function(app) {
-							if (app) {
-								var appUsers = _
-									.chain(app)
-									.get('users', [])
-									.map(function(val) {
-										return val.id;
-									})
-									.value();
-
-								if (app && app.allowed_users
-									&& (
-										(app.allowed_users === 'all')
-										|| (app.allowed_users === 'admins' && currentUser.priv_level === 'admin')
-										|| (app.allowed_users === 'specific' && _.includes(appUsers, currentUser.id))
-									)) {
-									return true;
-								}
-							}
-							return false;
+							var appAllowedUsers = _.get(app, 'allowed_users'),
+								areAllUsersAllowed = appAllowedUsers === 'all',
+								isAdminUserAllowed = appAllowedUsers === 'admins' && currentUser.priv_level === 'admin',
+								isCurrentUserAllowed = appAllowedUsers === 'specific'
+									&& _
+										.chain(app)
+										.get('users', [])
+										.some({ id: currentUser.id })
+										.value();
```

Comment:
Why not store this result in a variable as well, or use the same `lodash#include`  check than before? It might make it easier to read.

Also, in this case using `lodash#find` vs `lodash#some`  combined with `lodash#isUndefined` would make more sense (although it has the same effect in this case).

Reply:
Sorry, I missed this comments a couple of days. Sounds good. I went with the first approach, it will make it easy to read.